test/core/integration/Integration.t.sol

```diff
@@ -257,4 +268,133 @@ abstract contract Integration_Test is Base_Test {
             "settled status call return data"
         );
     }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                                  MIRROR-FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    function burn(uint256 streamId) internal {
```

Comment:
IMO these functions are overkill. If the `value` is 0, then it is equivalent to to not passing any value at all.

```solidity
lockup.burn{ value: 0 }(streamId);
```

is same as 

```solidity
lockup.burn(streamId);
```

Reply:
> Do you find it useful to test the ETH balance increase in all functions

No I don't. One concrete test for `withdrawFee` is enough imo as its a fairly simple logic.